README.md

```diff
@@ -80,18 +88,7 @@ or create an API key by adding the `--new` flag
 conduce-api config set api-key --new --user=valid_user --host=valid_host
 ```
 
-If you provide an API key for your default user and host you will never have to enter credentials to use that account as long as the API key is valid.
-
-## Features
-
-The CLI provides many high level functions.  These include:
-
-- Creating API keys
-- Adding and removing datasets
-- Creating a dataset from a CSV or JSON file
-- Listing datasets and other resources
-
-All CLI functionality is available through `conduce/api.py` and all API functionality should be usable with or without a `.conduceconfig` file.
+If you configure an API key for your default user and host you will never have to enter credentials to use that account as long as the API key is valid.
```

Comment:
We don't cover setting a default user in `.conduceconfig`.  Maybe this needs an update to.

Reply:
The original line was: 

If you provide an API key for your default user and host you will never have to enter credentials to use that account as long as the API key is valid.

The wording was confusing as I wondered whether providing an API key with a flag for instance would make it so that I never again had to preset my username, password, or environment, or if providing meant something more (like configuring)

My assumption is that this is incorrect. I would have to provide the API key every time instead of the username, password, and env OR configure it permanently. Thus I wanted to use "configure".

Reply:
Nevermind, I was mistaken.